src/tlo/methods/hiv.py

```diff
@@ -2434,6 +2439,12 @@ def apply(self, person_id, squeeze_factor):
         # Confirm that the person is diagnosed (this should not run if they are not)
         assert person["hv_diagnosed"]
 
+        # check whether person had Rx at least 3 months ago and is now due repeat prescription
+        # alternate routes into testing/tx may mean person already has recent ARV dispensation
+        if person['hv_date_last_ART'] >= (
+                self.sim.date - pd.DateOffset(months=self.module.parameters['dispensation_period_months'])):
+            return
```

Comment:
@tdm32-- I've also just spotted that we don't have an EXPECTED_APPT_FOOTPRINT on this HSI (!?!?!?!).
Is that what we want?
We should probably be explicit about it, if so.

Reply:
I think it should be set in line 2665:
    @property
    def EXPECTED_APPT_FOOTPRINT(self):

Reply:
Phew!! My little check didn't think about `properties`. :-)

Reply:
But, shouldn't the `return` to appointment appointment, still be `return <<blank_footprint>>`

Reply:
yes you're right, we should have a return blank footprint statement there if the condition returns true and the appt is aborted. I'll do it now